javascript/test/TripServiceSpec.js

```diff
@@ -0,0 +1,11 @@
+import assert from 'assert';
+import TripService from '../src/TripService';
+
+describe('TripService', () => {
+
+    it('should return no trips if users are not friends', () => {
+        const tripService = new TripService();
+        assert.equal(tripService.getTripsByUser('not a friend'), []);
+    });
+
+});
```

Comment:
We should not provide any tests. :) 

The objective of the exercise is that people retrofit the tests until the code is 100% covered and then refactor it to make it better.